### PR TITLE
[WIP] Create newsletter April 2022

### PR DESCRIPTION
**Pull request by aleesteele**
## Creating draft newsletter for April 2022

_Tentative release date: April 20_

**[Tasks before release](https://the-turing-way.netlify.app/community-handbook/newsletters/newsletters-process.html)**
- [x] Collect events from past month (mid-March 2022)
- [x] Review draft on HackMD (send me an email to review [asteele@turing.ac.uk](asteele@turing.ac.uk))
- [x] Post on Github
- [x] Review PR
- [ ] Publish on Tiny Letter
- [ ] Promote more broadly on Twitter, Turing Slack

### CODE REVIEW

**Comment by aleesteele** [comment]
https://github.com/carpentries-incubator/managing-computational-projects

**Comment by aleesteele** [comment]
https://www.gov.uk/government/publications/better-broader-safer-using-health-data-for-research-and-analysis/better-broader-safer-using-health-data-for-research-and-analysis

**Comment by aleesteele** [comment]
https://www.gov.uk/government/publications/better-broader-safer-using-health-data-for-research-and-analysis/better-broader-safer-using-health-data-for-research-and-analysis

**Comment by malvikasharan** [comment]
Anne, I think you and I did commits at the same time, so you might need to add the changes again. Sorry about that.
General comments are for fomatting - see: https://the-turing-way.netlify.app/community-handbook/newsletters/newsletters-style.html
- use line breaks 
- Use italicised _The Turing Way_
- Image titles should be added
- Use alt text to describe images
- Reduce the number of links you add - too many links flag newsletter as spam

**Comment by aleesteele** [comment]
Thank you so much, @malvikasharan! Thanks for flagging the formatting issues. I've added the alt texts and photo titles.

File: communications/newsletters/newsletter_33_Apr2022.md [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+
+**Chaired by Anne Lee Steele, this panel will feature insights from Dorothea Hug Peter, Elio Campitelli, Lilly Winfree, Gilbert Beyamba, and Ben Krikler.** These individuaals represent different communities with expertise and insights from organising and participating in international conferences.  
+[Sign up on the Eventbrite](https://www.eventbrite.co.uk/e/fireside-chat-making-virtual-participation-a-global-default-tbc-tickets-318719758377).
+
+## Sign up for our Book Dash 📚💨
+
+**The 7th [Book Dash](https://the-turing-way.netlify.app/community-handbook/bookdash.html) event will take place online from 16 to 20 May 2022.** Submit your application via this [Google form](https://forms.gle/uEbu1FJMmdfgGwvW6).
+The Deadline for submission is 21 April 2022.
+Please reach email me at [asteele@turing.ac.uk](mailto:asteele@turing.ac.uk) if you think you'll need need more time to complete the application.
```
Comment (Arielle-Bennett) [inline communications/newsletters/newsletter_33_Apr2022.md:36]: Either "please reach out" or "please email me"

File: communications/newsletters/newsletter_33_Apr2022.md [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+
+## Community News
+
+### Featured: UK's Goldacre review!
+
+![Alt:Screenshot of Goldacre report page and title](https://i.imgur.com/RFFH8iP.jpg)
+_The Goldacre Review was published on 7 April 2022._
+
+We are thrilled and honoured that [_The Turing Way_ was recently featured in the Goldacre Review](https://www.gov.uk/government/publications/better-broader-safer-using-health-data-for-research-and-analysis/better-broader-safer-using-health-data-for-research-and-analysis).
+You can read the [full report online](https://www.gov.uk/government/publications/better-broader-safer-using-health-data-for-research-and-analysis/better-broader-safer-using-health-data-for-research-and-analysis).
```
Comment (Arielle-Bennett) [inline communications/newsletters/newsletter_33_Apr2022.md:65]: Noting that Esther flagged that should probably only use one link as these are both the same...

File: communications/newsletters/newsletter_33_Apr2022.md [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+You can also sign up to host one of the future Collaboration Cafés on 4 May and 18 May 2022. We hope to see you there!
+Due to Book Dash in the third week of May, we won't be hosting a Collaboration Café on that week.
+
+#### Community Share-Outs on 20 May
+
+On 20 May 2022, on the last day of Book Dash, we will host two [Community Share Out and Contributor Celebrations Events](https://the-turing-way.netlify.app/community-handbook/bookdash/bookdash-events.html#community-share-outs-contributor-celebration)
+This is a public event for folks interested to learn about The Turing Way, seeing what recent updates have been made in the project and celebrating the incredible work of our contributors.
+You can sign up to attend one or both sessions on [Eventbrite](https://www.eventbrite.co.uk/e/the-turing-way-community-share-out-and-contributor-celebrations-may-2022-tickets-323373307267).
+Our Book Dash attendees will showcase their contributions to the book, share what they are working on and invite collaborations.
+We will also give an early preview of new set of illustrations as well as launch the new version of _The Turing Way_ book.
```
Comment (malvikasharan) [inline communications/newsletters/newsletter_33_Apr2022.md:59]:
```suggestion
### Community Share-Outs on 20 May

On 20 May 2022, the last day of Book Dash, we will host two [Community Share Out and Contributor Celebrations Events](https://the-turing-way.netlify.app/community-handbook/bookdash/bookdash-events.html#community-share-outs-contributor-celebration)
This is a public event for folks interested to learn about The Turing Way, seeing what recent updates have been made in the project and celebrating the incredible work of our contributors.
You are invited to attend one or both sessions, please sign up on [Eventbrite](https://www.eventbrite.co.uk/e/the-turing-way-community-share-out-and-contributor-celebrations-may-2022-tickets-323373307267).
We will also give an early preview of new set of illustrations as well as launch the new version of _The Turing Way_ book.
```

File: communications/newsletters/newsletter_33_Apr2022.md (hunk 2 above)
Comment (Arielle-Bennett) [reply communications/newsletters/newsletter_33_Apr2022.md:65]: Maybe would be worth writing one sentence on why the Turing Way was highlighted?

File: communications/newsletters/newsletter_33_Apr2022.md [hunk 4]
```diff
@@ -0,0 +1,259 @@
...
+![Alt:Screenshot of tweet by Lena](https://i.imgur.com/Lfr2AzN.png)
+_Lena's tweet about the new chapter!_
+
+Lena Karvovskaya recently published a new chapter in the Turing Way on [Peer Review](https://the-turing-way.netlify.app/communication/peer-review/peer-review-overview.html)!
+You can find [the new chapter in our Guide for Communication]((https://the-turing-way.netlify.app/communication/)).
+This new chapter includes helpful suggestions on what to do before accepting an invitation for peer review, and how to structure writing one.
+It also includes tips drawn from work by Rene Bekkers, called ["How to review a paper"](https://renebekkers.wordpress.com/2020/06/24/how-to-review-a-paper/).
+Thank you to Lena for writing, and to Vicky Hellon, Emma Karoune, and Achintya Rao for reviews!
+
+If you'd like to get involved with new and ongoing chapters, please get in touch with us: join a co-working hour, collaboration cafe, or message one of the team directly.
```
Comment (Arielle-Bennett) [inline communications/newsletters/newsletter_33_Apr2022.md:82]: Capitalise Collaboration Cafe

File: communications/newsletters/newsletter_33_Apr2022.md [hunk 5]
```diff
@@ -0,0 +1,259 @@
...
+
+- Arielle Bennett joined a Panel on Ethics, chaired by Laura Carter.
+Both are authors and mentors of several chapters in our [Guide for Ethical Research](https://the-turing-way.netlify.app/ethical-research/ethical-research.html?highlight=ethics).
+[Watch the recording here](https://www.youtube.com/watch?v=AfotGso8rDI)!
+
+![Alt:Screenshots of Slack posts and presentation titles](https://i.imgur.com/NYfMx7q.jpg)
+_Congratulations to the participants and presenters at #CW22!_
+
+- Achintya Rao and his team members developed a [Carpentries-style](https://carpentries.org/) lesson on R's bug tracking process, that was awarded third place during the hack day.
+- Malvika Sharan and her team members developed collaboration project idea, "Hi-Brid! Leaving no one behind in hybrid working!" that won second place during the Collaborative Idea's session.
```
Comment (Arielle-Bennett) [inline communications/newsletters/newsletter_33_Apr2022.md:136]: "developed the collaboration project idea"

File: communications/newsletters/newsletter_33_Apr2022.md [hunk 6]
```diff
@@ -0,0 +1,259 @@
...
+Alternatively, you can also [schedule time in her calendly](https://calendly.com/aleesteele/).
+
+### Research on Open Governance Models
+
+![Alt:Screenshot of Github pull request](https://i.imgur.com/F30aRgk.jpg)
+_Check out the Github issue to learn more about the research done so far on open governance models_
+
+Malvika and Anne are also co-leading an investigation into open research governance practices in the fifth cohort of [Open Life Science program](https://openlifesci.org/) (OLS-5).
+This research will inform the next stage of mature governance structure in _The Turing Way_.
+We would love to involve members of our community in this research as well as , please comment on [Pull Request #31 on OLS-5 Github](https://github.com/open-life-science/ols-5/issues/31) or reach out to us on Slack directly.
```
Comment (Arielle-Bennett) [inline communications/newsletters/newsletter_33_Apr2022.md:170]: "as well as...." phrase feels unfinished

**Review by Arielle-Bennett — COMMENTED** [review]
Some minor comments & typoes but otherwise looks good!

File: communications/newsletters/newsletter_33_Apr2022.md (hunk 5 above)
Comment (aleesteele) [reply communications/newsletters/newsletter_33_Apr2022.md:136]: Thanks for catching that, Arielle!

File: communications/newsletters/newsletter_33_Apr2022.md (hunk 6 above)
Comment (aleesteele) [reply communications/newsletters/newsletter_33_Apr2022.md:170]: Thanks for catching that1

File: communications/newsletters/newsletter_33_Apr2022.md [hunk 7]
```diff
@@ -0,0 +1,259 @@
+The Turing Way Newsletter: 20 April 2022
+
+# New chapters, sign up for book dash, the upcoming fireside chat, and (many) other community updates
```
[the remark this answers is not in the document]
Comment (malvikasharan) [reply communications/newsletters/newsletter_33_Apr2022.md:3]:
```suggestion
# New chapters, sign up for Book Dash, the upcoming Fireside Chat, and other community updates
```

File: communications/newsletters/newsletter_33_Apr2022.md [hunk 8]
```diff
@@ -0,0 +1,259 @@
...
+
+---
+
+# Upcoming Events
+
+## Tune into our Fireside Chat 🔥
+
+![](https://i.imgur.com/CQFTKn8.jpg)
+
+**On 29 April 2022, 15:00-16:30 UTC ([in your timezone](https://arewemeetingyet.com/UTC/2022-04-29/15:00)): the next fireside chat will take place on the topic: “Enabling Inclusive Conferencing in the 'Post-Pandemic Era'”.**
```
[the remark this answers is not in the document]
Comment (malvikasharan) [reply communications/newsletters/newsletter_33_Apr2022.md:24]:
```suggestion
**On 29 April 2022, 15:00-16:30 UTC ([in your timezone](https://arewemeetingyet.com/UTC/2022-04-29/15:00)): the next Fireside Chat will take place on the topic: “Enabling Inclusive Conferencing in the 'Post-Pandemic Era'”.**
```
```suggestion
**On 29 April 2022, 15:00-16:30 UTC ([in your timezone](https://arewemeetingyet.com/UTC/2022-04-29/15:00)): the next Fireside Chat will take place on the topic: “Enabling Inclusive Conferencing in the 'Post-Pandemic Era'”.**
```

File: communications/newsletters/newsletter_33_Apr2022.md [hunk 9]
```diff
@@ -0,0 +1,259 @@
...
+Over the past 2 and a half years, the COVID-19 pandemic has brought about a reckoning with how scientific research is done across international borders – and how scientific conferencing is done in turn.
+Alongside the shift to “working from home”, conferences also moved online.
+But as many events begin to move back to in-person events in the coming months and years, this Fireside chat brings together diverse voices from different projects and communities to talk more about the role of conferences in scientific collaboration, and the trade-offs involved in this shift back to in-person conferences.
```
[the remark this answers is not in the document]
Comment (malvikasharan) [reply communications/newsletters/newsletter_33_Apr2022.md:27]:
```suggestion
But as many events begin to move back to in-person events in the coming months and years, this Fireside Chat brings together diverse voices from different projects and communities to talk more about the role of conferences in scientific collaboration, and the trade-offs involved in this shift back to in-person conferences.
```

File: communications/newsletters/newsletter_33_Apr2022.md [hunk 10]
```diff
@@ -0,0 +1,259 @@
...
+
+**Chaired by Anne Lee Steele, this panel will feature insights from Dorothea Hug Peter, Elio Campitelli, Lilly Winfree, Gilbert Beyamba, and Ben Krikler.** These individuaals represent different communities with expertise and insights from organising and participating in international conferences.
```
[the remark this answers is not in the document]
Comment (malvikasharan) [reply communications/newsletters/newsletter_33_Apr2022.md:29]:
```suggestion
**Chaired by Anne Lee Steele, this panel will feature insights from Dorothea Hug Peter, Elio Campitelli, Lilly Winfree, Gilbert Beyamba, and Ben Krikler.** These individuals represent different communities with expertise and insights from organising and participating in international conferences.  
```

File: communications/newsletters/newsletter_33_Apr2022.md [hunk 11]
```diff
@@ -0,0 +1,259 @@
...
+Alongside the shift to “working from home”, conferences also moved online.
+But as many events begin to move back to in-person events in the coming months and years, this Fireside chat brings together diverse voices from different projects and communities to talk more about the role of conferences in scientific collaboration, and the trade-offs involved in this shift back to in-person conferences.
+
+**Chaired by Anne Lee Steele, this panel will feature insights from Dorothea Hug Peter, Elio Campitelli, Lilly Winfree, Gilbert Beyamba, and Ben Krikler.** These individuaals represent different communities with expertise and insights from organising and participating in international conferences.  
+[Sign up on the Eventbrite](https://www.eventbrite.co.uk/e/fireside-chat-making-virtual-participation-a-global-default-tbc-tickets-318719758377).
+
+## Sign up for our Book Dash 📚💨
+
+**The 7th [Book Dash](https://the-turing-way.netlify.app/community-handbook/bookdash.html) event will take place online from 16 to 20 May 2022.** Submit your application via this [Google form](https://forms.gle/uEbu1FJMmdfgGwvW6).
+The Deadline for submission is 21 April 2022.
```
[the remark this answers is not in the document]
Comment (malvikasharan) [reply communications/newsletters/newsletter_33_Apr2022.md:35]:
```suggestion
The deadline for submission is 21 April 2022.
```

these are my errors from editing! Thanks for catching :D

File: communications/newsletters/newsletter_33_Apr2022.md (hunk 4 above)
Comment (malvikasharan) [reply communications/newsletters/newsletter_33_Apr2022.md:82]:
```suggestion
If you'd like to get involved with new and ongoing chapters, please get in touch with us: join a co-working hour, Collaboration Cafe, or message one of the team directly.
```

File: communications/newsletters/newsletter_33_Apr2022.md (hunk 1 above)
Comment (malvikasharan) [reply communications/newsletters/newsletter_33_Apr2022.md:36]:
```suggestion
Please email me at [asteele@turing.ac.uk](mailto:asteele@turing.ac.uk) if you think you'll need need more time to complete the application.
```

File: communications/newsletters/newsletter_33_Apr2022.md (hunk 6 above)
Comment (malvikasharan) [reply communications/newsletters/newsletter_33_Apr2022.md:170]:
```suggestion
We would love to involve members of our community in this research, please comment on [Pull Request #31 on OLS-5 Github](https://github.com/open-life-science/ols-5/issues/31) or reach out to us on Slack directly.
```

Again that was me! I did some editing and for sure got distracted.

File: communications/newsletters/newsletter_33_Apr2022.md (hunk 5 above)
Comment (malvikasharan) [reply communications/newsletters/newsletter_33_Apr2022.md:136]:
```suggestion
- Malvika Sharan and her team members developed a collaborative project idea, "Hi-Brid! Leaving no one behind in hybrid working!" that won second place during the Collaborative Idea's session.
```

File: communications/newsletters/newsletter_33_Apr2022.md (hunk 2 above)
Comment (malvikasharan) [reply communications/newsletters/newsletter_33_Apr2022.md:65]:
```suggestion
_The Turing Way_ was highlighted in the context of addressing *a range of specific challenges around open working, the limits of open code, and the strong positive relationship between open working methods and innovative commercial activity in health data science*.
You can read the [full report online](https://www.gov.uk/government/publications/better-broader-safer-using-health-data-for-research-and-analysis/better-broader-safer-using-health-data-for-research-and-analysis).
```

Comment (malvikasharan) [reply communications/newsletters/newsletter_33_Apr2022.md:70]: Great idea Arielle, added a sentence.

**Review by malvikasharan — APPROVED** [review]

**Comment by aleesteele** [comment]
Thanks so much for reviewing, @malvikasharan and @Arielle-Bennett!

**Pull request merged**